packages/itwinui-react/src/core/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -114,70 +114,41 @@ type BreadcrumbsProps = {
  */
 const BreadcrumbsComponent = React.forwardRef((props, ref) => {
   const {
-    children: items,
-    currentIndex = items.length - 1,
+    children: childrenProp,
+    currentIndex: currentIndexProp,
     separator,
     overflowButton,
     className,
     ...rest
   } = props;
 
-  const [overflowRef, visibleCount] = useOverflow(items.length);
-  const refs = useMergedRefs(overflowRef, ref);
+  const items = React.useMemo(
+    () => React.Children.toArray(childrenProp),
+    [childrenProp],
+  );
+  const currentIndex = currentIndexProp || items.length - 1;
```

Comment:
It's confusing to set default values in the middle of the component definition; can we use `React.Children.count(childrenProp) - 1` when destructuring it?